runtime/lib/app-runtime.js

```diff
@@ -1012,13 +1011,6 @@ AppRuntime.prototype.multimediaMethod = function (name, args) {
     name, args)
 }
 
-/**
- * @private
- */
-AppRuntime.prototype.onGetPropAll = function () {
-  return {}
-}
```

Comment:
Not suggest removing this directly, could we use deprecation message instead for now?